src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md

```diff
@@ -0,0 +1,314 @@
+  <figure>
+    <img src="images/roboto-family.png" alt="Different members of the Roboto family">
+  </figure>
+</div>
+<div class="clearfix"></div>
+
+<div class="note"><strong>Note:</strong><span> Roboto Family designed by Christian Robertson.
+On the left, a specimen of Roboto. On the right different members of the Roboto family.</span></div>
+
+### Challanges for the Designer and Developer
+
+When a graphic designer prepares their work, they typically export the final
+artwork in a way that either embeds all of the fonts used, or they would provide
+an export archive that contains all of the assets used, a bit like a basic
+website. The cost to the designer depends on the suitability of the typeface for
+the rendering context  (is it legible at small sizes for example) and the
+license to use the fonts.
+
+On the web, we have both of the above for consideration, but also the associated
+bandwidth cost. For every member of the family or style used in our designs, we
+require that font file to be downloaded by the reader before the text is
+rendered. Just including the regular and bold styles plus their italic
+variations can amount to 500k or more of data. This has been a sticking point
+for web designers, and developers as a customized typography experience can come
+at a cost. This is even before we have dealt how the font renders and the
+patterns we are going to use (FOIT and FOUT).
+
+## Anatomy of a variable font A variable font is a collection of masters, with
+one Default master, (usually the Regular font), and multiple registered “Axes”,
+such as the Weight axis, which would include the Regular and Black weights of
+the typeface. These individual styles along the axes are called instances.
+<strong>Width</strong>, <strong>Optical Size</strong> and <strong>Slant</strong>
+are other examples, where the instances along those axes represent the range of
+that axes, 
+([OT spec](https://docs.microsoft.com/en-gb/typography/opentype/spec/dvaraxisreg){: .external}).
+
+
+<figure>   <img src="images/Amstelvar-Alpha-example.png" alt="Example of the
+Weight Axes for the typeface Amstelvar">   <figcaption>Typeface Amstelvar,
+designed by David Berlow, type designer and typographer at Font
+Bureau.</figcaption> </figure>
+
+All of these axes share the Default, and by traversing between the masters of
+each axis we get to explore multiple options of the font, not just the specific
+axes, but their combinations. For example, variable font Amstelvar has three
+masters for its <strong>Weight axis</strong>, Regular, which is at the center of
+the Weight axis scale, and two masters, thinner and bolder at the opposite ends
+of the axis. Between these, there are potentially 200 variations on the axis
+that give the designer or developer an array of weights to choose from and
+explore.
+
+[Amstelvar](https://github.com/TypeNetwork/Amstelvar){: .external} also has
+[three masters for its Width axis, Regular, which is at the center of the axis
```

Comment:
All of these lines start with `[` - did you mean to do that?

Reply:
oops good catch